swift/WorkflowUI/Sources/Screen/AnyScreen/AnyScreen.swift

```diff
@@ -19,24 +19,22 @@
 import UIKit
 
 public struct AnyScreen: Screen {
```

Comment:
Happy to see this collapse into just a passthrough for the view controller description